lib/flow.js

```diff
@@ -33,6 +33,12 @@ Flow.prototype.setSteps = function (steps) {
   this._steps = steps;
 };
 
+Flow.prototype.mergeSteps = function (steps) {
+  // better to use _.defaults than _.merge which merge steps arrays which
+  // is not waht is wanted here.
```

Comment:
sorry! a non native english speaker should not try to write something intelligent at night :)